cves/CVE-2011-3192.yml

```diff
@@ -32,7 +32,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+    This vulnerability involves resource exhaustion leading to a denial of service
+    due to improper handling of overlapping byte ranges in http requests. The eventual fix nearly 7 years later involved reducing the 
+    memory used by these requests, and they made it so that byteranges were 
```

Comment:
**Grammar**
memory used by these requests `such that byte ranges` were ....

Reply:
Fixed